priv/app/call_tracer.jsx

```diff
@@ -158,26 +234,38 @@ export default class CallsTracer extends React.Component {
             <div className="form-group">
               <div className="input-group">
                 <div className="input-group-addon">Treshold</div>
+                <span className={thresholdClass}>
                 <input ref="thresholdInput" type="text" className="form-control"
-                       id="tresholdInput" placeholder="100"/>
+                  id="tresholdInput" placeholder={this.props.defaultThreshold}
```

Comment:
For `placeholder` I'll put a prompt text e.g. `Insert here a threshold e.g. ${this.props.defaultThreshold}`.